### DISCUSSION TITLE
Fall back to scraping when api max quota is hit

### DISCUSSION THREAD

**Discussion by Oondanomala** [ThioJoe/YT-Spammer-Purge#417] [Ideas]
When the api max quota is hit the app could use the same system it uses for community tabs to keep scanning comments and saving them in a log file to report later.

**Comment by Kenshin9977** [comment]
Might be pretty cumbersome to develop and maintain as there are many anti-scraping tools that Youtube may use

**Reply by Oondanomala** [reply]
I don't really have any experience with this, but looking at yt-dlp (which can also scrape comments) it doesn't look like it needs much effort to maintain. (latest fix was on 18 October and it was just an android youtube layout change)
And more importantly AFAIK this is also used for community tabs so most of the code is already there.
Who am i to know tho 😄

**Reply by Kenshin9977** [reply]
Well in this case yeah it should be pretty easy. We could even use yt-dlp to do that and just use its comments scraping functions
